backend/src/api/candidates.js

```diff
@@ -37,6 +38,22 @@ router.post(
   })
 )
 
+router.get(
+  '/initialize-git',
+  errorWrap(async (req, res) => {
+    const candidates = await Candidate.find()
+    candidates.map(async candidate => {
+      if (!candidate.github) {
+        await Candidate.findByIdAndUpdate(candidate._id, { githubContributions: 'N/A' })
+      } else {
+        let contributions = await getGithubContributions(candidate.github)
```

Comment:
getGithubContributions is not asynchronous

Reply:
It is because it's making a request to scrape the website, if you remove the await in you get this in the database : ```"githubContributions":"[object Promise]"```

Reply:
ah my bad